client/modules/food/components/packing/PackingList.js

```diff
@@ -174,6 +179,7 @@ class PackingList extends Component {
               onSelect: this.handleSelect,
               onSelectAll: this.handleSelectAll,
               selected,
+              unselectable: unselectable,
```

Comment:
Can be just written as `unselectable` and expands to `unselectable: unselectable`, like `selected` in the line above.

Reply:
Thanks, I’ll fix them.